app/models/service_ansible_playbook.rb

```diff
@@ -162,12 +134,15 @@ def use_default_inventory?(hosts)
     hosts.blank? || hosts == 'localhost'
   end
 
+  def hosts_array(hosts_string)
+    hosts = 'localhost' if use_default_inventory?(hosts_string)
+    hosts.split(',')
```

Comment:
Should this also do a `.map(&:strip)` in case the user puts `, `

Reply:
Yeah, probably also a `.delete_blanks` for good measure. I'll add specs for stuff like `"localhost, 192.168.12.1,  ,"`